src/classes/typeValidation.ts

```diff
@@ -0,0 +1,134 @@
+//this class validates the MAIN input types
+export class Validation
+{
+    constructor(){}
+
+    // ID
+    isID(id: number): boolean
+    {
+        //id &&
+        if( typeof id === 'number')
+        {
+            return true;
+        }
+        else
+        {
+            console.log("Id not valid");
+            return false;
+        }
+    }
+
+    isString(input: string): boolean
+    {
+        if(input != '' && typeof input === 'string')
```

Comment:
An empty string is not considered a string in this function. Either make this clear in the function name or consider an empty string valid